cli/azd/internal/appdetect/spring_boot.go

```diff
@@ -263,22 +350,53 @@ func detectStorageAccountAccordingToSpringCloudStreamBinderMavenDependencyAndPro
 			}
 		}
 		if containsInBindingName != "" {
-			containerNamePropertyMap := make(map[string]string)
-			for key, value := range springBootProject.applicationProperties {
-				if strings.HasSuffix(key, targetPropertyName) {
-					containerNamePropertyMap[key] = value
-				}
-			}
-			newDep := AzureDepStorageAccount{
-				ContainerNamePropertyMap: containerNamePropertyMap,
-			}
-			azdProject.AzureDeps = append(azdProject.AzureDeps, newDep)
-			logServiceAddedAccordingToMavenDependencyAndExtraCondition(newDep.ResourceDisplay(), targetGroupId,
-				targetArtifactId, "binding name ["+containsInBindingName+"] contains '-in-'")
-			for property, containerName := range containerNamePropertyMap {
-				log.Printf("  Detected Storage container name: [%s] for [%s] by analyzing property file.",
-					containerName, property)
-			}
+			detectStorageAccountAccordingToProperty(azdProject, springBootProject.applicationProperties,
+				targetGroupId, targetArtifactId, targetPropertyName,
+				"binding name ["+containsInBindingName+"] contains '-in-'")
+		}
+	}
+}
+
+func detectStorageAccountAccordingToSpringIntegrationEventhubsMavenDependencyAndProperty(
+	azdProject *Project, springBootProject *SpringBootProject) {
+	var targetGroupId = "com.azure.spring"
+	var targetArtifactId = "spring-cloud-azure-starter-integration-eventhubs"
+	var targetPropertyName = "spring.cloud.azure.eventhubs.processor.checkpoint-store.container-name"
+	if hasDependency(springBootProject, targetGroupId, targetArtifactId) {
+		detectStorageAccountAccordingToProperty(azdProject, springBootProject.applicationProperties,
+			targetGroupId, targetArtifactId, targetPropertyName, "")
+	}
+}
+
+func detectStorageAccountAccordingToSpringMessagingEventhubsMavenDependencyAndProperty(
+	azdProject *Project, springBootProject *SpringBootProject) {
+	var targetGroupId = "com.azure.spring"
+	var targetArtifactId = "spring-messaging-azure-eventhubs"
+	var targetPropertyName = "spring.cloud.azure.eventhubs.processor.checkpoint-store.container-name"
+	if hasDependency(springBootProject, targetGroupId, targetArtifactId) {
+		detectStorageAccountAccordingToProperty(azdProject, springBootProject.applicationProperties,
+			targetGroupId, targetArtifactId, targetPropertyName, "")
+	}
+}
+
+func detectStorageAccountAccordingToProperty(azdProject *Project, applicationProperties map[string]string,
+	targetGroupId string, targetArtifactId string, targetPropertyName string, extraCondition string) {
```

Comment:
Rename `targetPropertyName` to `targetPropertyNameSuffix`. Same to other places.

Reply:
Updated.